newIDE/app/src/MainFrame/EditorContainers/HomePage/EducationMarketingSection/index.js

```diff
@@ -203,53 +216,88 @@ const EducationMarketingSection = ({
+                    The Education subscription gives access to GDevelop's Game
+                    Development curriculum. Co-created with teachers and
+                    institutions, it’s a ready-to-use, proven way to implement
+                    STEM in your classroom.
                   </Trans>
                 </Text>
+                <Spacer />
+                <ResponsiveLineStackLayout noColumnMargin noMargin>
+                  <div style={styles.buttonContainer}>
+                    <FlatButton
+                      primary
+                      fullWidth
+                      onClick={() => {
+                        Window.openExternalURL('https://gdevelop.io/education');
+                      }}
+                      label={<Trans>Learn more</Trans>}
+                    />
+                  </div>
+                  <div style={styles.buttonContainer}>
+                    <RaisedButton
+                      primary
+                      fullWidth
+                      icon={<ShinyCrown fontSize="small" />}
+                      onClick={() => {
+                        openSubscriptionDialog({
+                          filter: 'education',
+                          analyticsMetadata: {
+                            reason: 'Callout in Classroom tab',
+                          },
+                        });
+                      }}
+                      label={<Trans>Subscribe to Edu</Trans>}
+                    />
+                  </div>
+                </ResponsiveLineStackLayout>
               </ColumnStackLayout>
-              <div style={styles.buttonContainer}>
-                <RaisedButton
-                  primary
-                  fullWidth
-                  onClick={() => {
-                    openSubscriptionDialog({
-                      filter: 'education',
-                      analyticsMetadata: { reason: 'Callout in Classroom tab' },
-                    });
-                  }}
-                  label={<Trans>Get the Edu subscription</Trans>}
-                />
-              </div>
             </ResponsiveLineStackLayout>
           </div>
         </CardWidget>
       </SectionRow>
       <SectionRow>
+        <div
+          style={{
+            ...styles.freeTag,
+            color: gdevelopTheme.statusIndicator.success,
+            borderColor: gdevelopTheme.statusIndicator.success,
+          }}
+        >
+          <Text noMargin color="inherit">
+            <Trans>Free</Trans>
+          </Text>
+        </div>
         <Text size="section-title">
-          <Trans>Teaching resources included!</Trans>
+          <Trans>Get our teaching resources</Trans>
         </Text>
         <Text>
           <Trans>
-            The Education subscription gives access to GDevelop’s Game
-            Development curriculum. Co-created with teachers and institutions,
-            it’s a helpful guide to your STEM related courses.
+            We’ll send to your email:
```

Comment:
We might need to `<Trans>` each item? Unsure how it will work on Crowdin, but we can try.

Reply:
So ignore this comment for now